### PR TITLE
Add pull approve functionality to repo. Fixes #866.

### LINKED ISSUE #866: Ensure only code approved by appropriate reviewers gets merged into the DCP
We will use a service/tool called "Pull Approve" to achieve this (https://www.pullapprove.com/).

Overview:
The DevSecOps team are going to sign up for a pullapprove account.
[PullApprove Overview](https://www.pullapprove.com/docs/)
[Enabling required status checks - GitHub](https://help.github.com/en/articles/enabling-required-status-checks)

### PR DESCRIPTION
**Pull request by malloryfreeberg**
### Release notes

- No schema changes.
- Added .pullapprove.yml to top level directory. Fixes #866.

### Reviews requested

- Need 2 Reviewers to approve because this is a new feature
- Last Reviewer to review/approve, merge changes by **Wednesday June 16**

### To test the new feature
1. Go to any open PR (or make a dummy PR).
1. Under the "Files changed" tab, click "Review changes".
1. In the comment section, type `/pullapprove-test`
1. Under this line, copy/paste the entire contents of the .pullapprove.yml  file inside a code block
Screen Shot 2019-06-05 at 10 19 54
1. Click "Submit review"
1. The text will appear as a comment on the PR, followed by an automatically generated comment from the pullapprove bot. Click the bot comment to see the results of what the pullapprove feature would do if it was enabled.

### CODE REVIEW

**Comment by malloryfreeberg** [comment]
@ESapenaVentura looks like the test worked for you - nice!

File: .pullapprove.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+  metadata-updates-patch:
+    conditions:
+    - "'*travis*' in statuses.succeeded"
+    - "base.ref == 'develop'"
+    - "'patch' in body" # patch change is indicated in body of PR
+    reviewers:
+      users:
+      - malloryfreeberg
+      - malloryfreeberg
```
Comment (daniwelter) [inline .pullapprove.yml]: why is everyone's name in here twice?

Comment (malloryfreeberg) [reply .pullapprove.yml]: Given that @cjvillar and @willrockout are not full time on HCA, I wanted to weight the rest of us to be chosen "randomly" more often. I'm not actually sure my hack works :) We'll see once pullapprove starts to be implemented and how often people are chosen.

**Comment by malloryfreeberg** [comment]
@daniwelter please merge when you can :)

**Pull request merged**